django_cloud_deploy/skeleton/utils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions related to Django project source files."""
+
+import os
+import re
+
+
+class ProjectContentError(Exception):
+    """An error thrown when Django project name cannot be determined."""
+
+
+def get_django_project_name(django_directory_path: str):
+    """Returns Django project name given a Django project directory.
+
+    In manage.py, there is a line as the follows:
+    "os.environ.setdefault('DJANGO_SETTINGS_MODULE',
+                           '{{ project_name }}.settings')"
+
+    We can determine the project name from this line.
+
+    Args:
+        django_directory_path: Absolute path of django project directory.
+
+    Raises:
+        ProjectContentError: When the function cannot find Django project name.
```

Comment:
This can raise another error, if on line 39 doesn't have manage.py in the path.

Reply:
Done.